### DISCUSSION TITLE
BIG library of musicxml files for free

### DISCUSSION THREAD

**Discussion by lucbonnin** [w3c-cg/musicxml#458] [Show and tell]
Hi folks,
We were so frustrated to not being able to access to all Mozart, Chopin or Beethoven for free that we decided to build a big database of free musicxml files (public domain).
And we bring it to the world **100% for free** as this human being knowledge should not be monetized at all in our opinion.
Feel free to use one or more of the musicxml files from 11k files on 
www.playthatsheet.com

Lot of classical music, religious music, turkish traditional makam, traditional folk and more...

One more thing, we add some pretty cool features so you can :
- display music sheet in 4 formats (music sheets, tablatures,text mode, multiple line or one infine line)
- change tempo
- transpose 
- play in audio
- scroll automatically as you play from your microphone (great pitch detection algorithm behind the scene)
- and send your own musicxml file to use it directly

**Comment by webern** [comment]
Would you consider publishing the files in a GitHub repository or something? So someone can pull them all in and use for a test suite?

**Reply by infojunkie** [reply]
You may be aware of this, but Lilypond has a MusicXML test suite https://github.com/lilypond/lilypond/tree/master/input/regression/musicxml

**Comment by ahankinson** [comment]
Did you encode all the files yourself? Where did you get the transcriptions?

**Reply by JDLH** [reply]
And a follow-up question: what is the licensing of these MusicXML files? There are at least two sets of copyrights to consider (as far as I, a non-lawyer, understand it): 1. the copyrights of the original composer, lyricist, editor etc. of the music, and 2. your copyright as transcriber. The licensing will be a function of both.  The best kind of license is, "in the public domain", or [cc0](https://creativecommons.org/choose/zero/).

**Reply by mscuthbert** [reply]
at least some of them seem to come from Wikifonia archive, which was licensed at the time they were put online but taken down because the license expired.  I use them in my research because I received them when they were licensed but I don't redistribute them because I don't have the rights.  Maybe PTS was able to reacquire the rights?

**Reply by napulen** [reply]
I checked the metadata of a few `.musicxml` files. They do not seem to have any provenance information (or they had it removed) beyond a reference to the PTS site. 

How would you know they come from Wikifonia @mscuthbert? File naming convention?

**Reply by mscuthbert** [reply]
One of the files (a piece I knew from Wikifonia) has it as a source url in the miscellaneous metadata field.  It was also the only leadsheet source I knew of that had so much popular music encoded as melody + chords.  I looked at those pieces a lot in writing https://emusicology.org/article/view/5889/4974

**Reply by lucbonnin** [reply]
Hi folks,
Some of them comes from an old backup of wikifonia but vast majority of them came from various R&D studies from various countries in the world and the vast majority of them are copyright free (public domain).
Some of them may have a copyright and if there is any claim, I will remove them from playthatsheet.

**Reply by ahankinson** [reply]
It would be great to have provenance information for each file, for a couple reasons:

1. As data, it's important to know where something comes from. Lots of encodings out there are of dubious quality, and people have come to learn that some sources are better than others. Also, whether the editor has taken any "liberties" in transcribing (simplifying, transposing, etc.) Being transparent about where you got the sheet music is quite useful, and has few downsides. 
2. Attribution of the "encoder" is important. Transcribing sources is very labour intensive. Although the **music content** is in the public domain, this doesn't meant that the work that went in to creating the digital files is "free." You should keep any attributions or source information, if available -- even if those attributions aren't in the actual files themselves. If the encodings were produced as part of a research grant or scholarly work, then it's usually pretty important that they are able to measure the "impact" of their work.

**Reply by lucbonnin** [reply]
It is part of my TODO to add this info available (data sources + software used to encode the musicxml and so on).
I'm currently not able to work on this project but when I will, it will be adde

**Comment by webern** [comment]
The link in the original post is a 404: https://github.com/w3c/musicxml/discussions/url

That doesn't seem like the intended link

**Reply by JDLH** [reply]
Good catch! The link text says `www.playthatsheet.com` . The link _destination_ in the original post is completely different, as you point out. I did not notice that, because I just typed the domain name into my browser's location bar, and it worked. I don't want to link to the original post's website, because I now have doubts that all their content is "100% free". I suggest that anyone who wants to check out the website, try copying and pasting domain name like I did.

**Reply by lucbonnin** [reply]
good catch. I've changed the link. Weird thing as it seems an issue of the github chat. Anyway, playthatsheet is not a scam at all. I've created it for my personal use, then to offer it to other musician as I was astonished when I discovered how hard it was to gather lot of musicxml in public domain for free. So I decided to do it and share it :)
+ of course my initial aim of learning to read music without solfege for saxophone (at least for my current level of usage)
